frontend/src/libs/components/card/card.tsx

```diff
@@ -21,36 +23,51 @@ const Card: React.FC<Properties> = ({
   isActive = false,
   iconName,
   iconColor = 'currentColor',
+  iconRight,
+  onIconClick,
 }) => {
   const hasNoImageOrIcon = !imageUrl && !iconName;
   const hasImage = Boolean(imageUrl);
   const hasIcon = Boolean(iconName);
 
   return (
-    <button
-      className={getValidClassNames(
-        styles['item'],
-        isActive && styles['selected'],
-        hasNoImageOrIcon && styles['no-image'],
-      )}
-      onClick={onClick}
-    >
-      {!hasNoImageOrIcon && (
-        <div className={styles['image-container']}>
-          {hasImage && (
-            <div className={styles['image-placeholder']}>
-              <img src={imageUrl} alt="not found" className={styles['image']} />
-            </div>
-          )}
-          {hasIcon && (
-            <div className={styles['icon-background']}>
-              <Icon name={iconName as IconName} color={iconColor} />
+    <>
+      <button
```

Comment:
Remove useless wrapper with fragment

Reply:
@v0ldemar01 without it i ca`nt return two buttons as one component

Reply:
Got it

Reply:
Got it